bundle-size/api.js

```diff
@@ -454,13 +468,21 @@ exports.installApiRouter = (app, router, db, githubUtils) => {
 
   router.post('/commit/:headSha/report', async (request, response) => {
     const {headSha} = request.params;
-    const {baseSha, bundleSizes} = request.body;
+    // mergeSha is new, and not all reports will have it.
```

Comment:
Add a TODO (you can assign this to me) to make this non-optional after a month or so